app/pii/pii_base.py

```diff
@@ -115,7 +115,8 @@ def __new__(cls, value: str, is_encrypted: bool = False) -> 'Pii':
 
         # Encrypt the value
         if is_encrypted and len(value) > 50:
-            # Workaround until all existing notification tasks and retries have been drained, then remove the len check
+            # Workaround until all existing notification tasks and retries have been drained.
+            # Then remove the length check.
             encrypted = value
```

Comment:
Do you know why this is necessary?

Reply:
No, I don't know.  Kyle added that as a hotfix on July 8th.